ckanext/dalrrd_emc_dcpr/assets/css/dalrrd-emc-dcpr.css

```diff
@@ -731,6 +731,11 @@ footer {
   padding-bottom: 30px;
 }
 
```

Comment:
for this css i need to see a visual representation to get a glimpse, nevertheless it's trivial